common/source/docs/common-jae-jfb110.rst

```diff
@@ -0,0 +1,314 @@
++--------+---------------------+--------------------------+
+| 29     | PWM OUT 6           | GPIO 55                  |
++--------+---------------------+--------------------------+
+| 30     | PWM OUT 7           | GPIO 56                  |
++--------+---------------------+--------------------------+
+| 31     | PWM OUT 8           | GPIO 57                  |
++--------+---------------------+--------------------------+
+| 32     | GND                 | for PWM OUT              |
++--------+---------------------+--------------------------+
+
+103 port pin assignments
+========================
+
+.. image:: ../../../images/jae-jfb110-103-pins.png
+    :target: ../_images/jae-jfb110-103-pins.png
+    :width: 450px
+
++--------+---------------------+--------------------------+
+| Pin No | Name                | Details                  |
++--------+---------------------+--------------------------+
+| 1      | GND                 | For PWM OUT              |
++--------+---------------------+--------------------------+
+| 2      | GND                 | For USB                  |
++--------+---------------------+--------------------------+
+| 3      | USB OTG FS DM       | USB port                 |
++--------+---------------------+--------------------------+
+| 4      | USB OTG FS DP       | USB port                 |
++--------+---------------------+--------------------------+
+| 5      | PWM OUT 9           | GPIO 58                  |
++--------+---------------------+--------------------------+
+| 6      | PWM OUT 10          | GPIO 59                  |
++--------+---------------------+--------------------------+
+| 7      | PWM OUT 11          | GPIO 60                  |
++--------+---------------------+--------------------------+
+| 8      | PWM OUT 12          | GPIO 61                  |
++--------+---------------------+--------------------------+
+| 9      | PWM OUT 13          | GPIO 62                  |
++--------+---------------------+--------------------------+
+| 10     | PWM OUT 14          | GPIO 63                  |
++--------+---------------------+--------------------------+
+| 11     | PWM OUT 15          | GPIO 64                  |
++--------+---------------------+--------------------------+
+| 12     | PWM OUT 16          | GPIO 64                  |
++--------+---------------------+--------------------------+
+
+RC Input
+========
+
+RC input is configured on the port marked DSM/SBUS RC. This connector supports all RC protocols. Two cables are available for this port. To use software binding of Spektrum satellite receivers you need to use the Spektrum satellite cable.
+
+PWM Output
+==========
+
+The JFB-110 supports up to 16 PWM outputs. These are directly attached to the STM32H755
+
+- PWM 1 and 2 in group1 (TIM15)
+- PWM 3 and 4 in group2 (TIM3)
+- PWM 5, 11 ,12 and 13 in group3 (TIM4)
+- PWM 6 ,9 and 10 in group4 (TIM1)
+- PWM 7 ,8 and 15 in group5 (TIM5)
+- PWM 14 and 16 in group6 (TIM12)
+
+Battery Monitors
```

Comment:
we put the defaults here since they are in hwdef and a power brick is included with the autopilot for the first PM

Reply:
sorry I see no change here

Reply:
Hi, I don't really know what to do here I'm afraid..

Reply:
I will fix it...

Reply:
OK, thanks you so much and sorry for being somewhat helpless!